docs/userguides/testing.md

```diff
@@ -27,12 +27,10 @@ when it is the owner of the contract making the transaction.
 This is an example of how that test may look:
 
 ```python
-def test_is_owner(my_contract, owner, other):
+def test_is_owner(my_contract, owner, receiver):
```

Comment:
The example should highlight that the account is just anyone except the owner

```suggestion
def test_is_owner(my_contract, owner, not_owner):
```

Reply:
Updated as part of dee36a56